src/containerbuddy/etcd.go

```diff
@@ -0,0 +1,240 @@
+package main
```

Comment:
`discovery_test.go` really just tests the Consul service interface. We should move most or all of that into `consul_test.go` and include `etcd_test.go`.